composer.json

```diff
@@ -15,7 +15,7 @@
         }
     ],
     "require": {
-        "php": "^7.4|^8.0",
+        "php": "~7.4.0|~8.0.0|~8.1.0",
```

Comment:
Why do you limit it to 8.1? 

Basically this package was not broken in 8.1 afaics, it's just fixing some deprecations. So it might very well work for 8.2 too, we don't really know, it's the big debate on how strict you want to be on future versions, but I don't see why we should change this? It will only require us to change this package every time we upgrade php.

Reply:
> Basically this package was not broken in 8.1 afaics, it's just fixing some deprecations.

Since some deprecations may actually lead to fatal errors, my guiding principle is to consider code with deprecations as broken, and with that in mind, I committed this proposal to make the PHP version compatibility more explicit.

This versioning restriction can be seen in some PHP packages, especially in Laminas, where they use this strategy to limit the amount of downstream issues upon adoption of newer PHP versions due deprecations.

[Here's an example for reference](https://github.com/mezzio/mezzio/pull/96)

Reply:
Yep, I understand the tradeoff, a lot of projects choose to not restrict it, here are some examples for reference in [doctrine/orm](https://github.com/doctrine/orm/blob/2.11.x/composer.json#L23) or [phpunit](https://github.com/sebastianbergmann/phpunit/blob/master/composer.json#L25).

Reply:
> a lot of projects choose to not restrict it

Most of the projects did this by default. but after the last set of PHP upgrades the strategy of being more restrictive has been steadily adopted.

I'm also fine dropping it in our case since we're the only users of this fork. what would you prefer @MarijnKoesen ?